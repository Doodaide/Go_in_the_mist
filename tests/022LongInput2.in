who
place P17
place H10
place Q12
hist
history
place J8
place K15
who
place I9
place I11
view
place G11
place O7
place o7
place B7
place B7
who
place D14
place H17
place M18
place K7
