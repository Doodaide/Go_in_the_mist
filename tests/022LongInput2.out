B
Invalid!
P17H10Q12
W
S9,.#..xxx....xxx....xxx....xxx....xxx....xxx....xxx
Invalid coordinate
Occupied coordinate
B
White wins!
P17H10Q12J8K15I9I11G11O7B7D14H17M18K7
Thank you for playing!
